File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGJustify.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGJustify {
    YGJustifyFlexStart,
    YGJustifyCenter,
    YGJustifyFlexEnd,
    YGJustifySpaceBetween,
    YGJustifySpaceAround,
    YGJustifySpaceEvenly;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGJustify forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGJustify.kt
package app.cash.redwood.yoga.internal.enums

enum class YGJustify {
    YGJustifyFlexStart, YGJustifyCenter, YGJustifyFlexEnd, YGJustifySpaceBetween, YGJustifySpaceAround, YGJustifySpaceEvenly;

    fun getValue(): Int {
        return ordinal
    }

    companion object {

        fun forValue(value: Int): YGJustify {
            return values()[value]
        }
    }
}
